src/background/js/app.js

```diff
@@ -138,7 +165,7 @@ gApp.run(function ($rootScope, $location, $http, $timeout, ProfileService, Setti
                         }
 
                         $http.get(apiBaseURL + "account").success(function (data) {
-                            $rootScope.profile.user = data;
+                            //$rootScope.profile.user = data;
```

Comment:
It was still used at one place in `form.html` and `bg.html`; I modified them (not used anywhere else according to grep).

I changed all `profile.user` by `account` since I found it confusing (that `profile.user` was referring to the `account`, and was filled by `AccountService` instead of `ProfileService`); also, instead of loading it in and from the `rootScope`, I use the `AccountService` as much as possible.

No probs for changing it back if you'd rather have it the way it was !